clusterloader2/pkg/flags/flags.go

```diff
@@ -25,12 +25,10 @@ import (
 	"time"
 
 	"github.com/spf13/pflag"
-	"k8s.io/klog/v2"
 )
 
 func init() {
 	pflag.CommandLine = pflag.NewFlagSet(os.Args[0], pflag.ContinueOnError)
-	klog.InitFlags(nil)
```

Comment:
Importing a package should never unconditionally change the global flags. That needs to be under control of `main()`.

Reply:
I understand that this is the less convenient in unit tests. I am proposing https://github.com/kubernetes/kubernetes/tree/master/test/utils/ktesting as a solution for logging in unit tests, but it's currently only available inside Kubernetes.